contracts/acl/ACL.sol

```diff
@@ -156,6 +157,30 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
         _setPermissionManager(address(0), _app, _role);
     }
 
+    /**
+    * @notice Lock non-existing yet `_role` in `_app`, so no modification can be made to it (grant, revoke, permission manager)
+    * @param _app Address of the app in which the permission is being locked
+    * @param _role Identifier for the group of actions being locked
+    */
+    function createLockedPermission(address _app, bytes32 _role)
+        external
+        auth(CREATE_PERMISSIONS_ROLE)
+    {
+        _setPermissionManager(LOCK_ENTITY, _app, _role);
```

Comment:
This should also check `require(getPermissionManager(_app, _role) == address(0));`

Reply:
Good catch, thanks! I accidentally lost it during the split in last commit: https://github.com/aragon/aragonOS/pull/429/commits/9a89419c3cd053b444762de64ee1310ab044d536#diff-325598653bbd8046a79c33ddabaa86e1L170
Thinking about it, as the same condition is being used in `_createPermission`, and for consistency with some of the recent changes for aragon OS 4 / audit freeze, should I move that into a modifier and use it in external functions instead of that internal one?

Reply:
Really like https://github.com/aragon/aragonOS/pull/429/commits/28df834b7b932499dc6992626474405af49870ec!